ruby/src/main/java/com/google/protobuf/jruby/RubyMessage.java

```diff
@@ -860,6 +860,7 @@ protected DynamicMessage build(ThreadContext context, int depth, int recursionLi
               this.builder.getDescriptorForType().findFieldByName(TYPE);
           if (typeBytesSymbol.equals(fields.get(enumFieldDescriptorForType))) {
             isDefaultStringForBytes = true;
+                    }
```

Comment:
Specifically, this additional `}` seems to be the culprit

Reply:
Done.